Adds fourth batch of 4.8 RN bug doc text (#34257)
openshift/openshift-docs: release_notes/ocp-4-8-release-notes.adoc

* Parsing undefined values results in a not a number (NaN) exception and the *Chart* tooltip shows a box with no values. Specify a start date when fetching data to ensure that the results are synced and enure undefined values are not parsed so that the *Chart* toolip shows correct values. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906304[*BZ#1906304*])

* The input labels in the *Manage Columns* modal were not clickable buttons, so you could not click them to manage the columns. With this bug fix, the labels are now clickable buttons that you can use to manage columns. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1908343[*BZ#1908343*])

* CSI provisioners were not listed when creating a storage class on the Google Cloud Platform. With this bug fix, the issue is resolved. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1910500[*BZ#1910500*])

> * Previously if when using the web console, the user clicks into a *Cluster Role* from *User Management -> Roles* list view, the back link from the details page is *Cluster Roles* which provides a generic list view of *Cluster Roles*. This caused backward web console navigation to redirect to the incorrect page. The fix is for the back link to direct the user to the *Role/Bindings* list view from the *Cluster Role/RoleBinding* details page. This allows the user to correctly navigate backward in the web console.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1915971[*BZ#1915971*])

Reviewer: ```suggestion
* Previously, if the user clicked into a *Cluster Role* from the *User Management -> Roles* list view, the back link from the details page is *Cluster Roles*, which provides a generic list view of *Cluster Roles*. This caused backward web console navigation to redirect to the incorrect page. With this release, the back link directs the user to the *Role/Bindings* list view from the *Cluster Role/RoleBinding* details page. This allows the user to correctly navigate backward in the web console.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1915971[*BZ#1915971*])
```

> * Previously, *Created date time* was not displayed in a format that a person could read. This meant it was difficult to understand and use the time shown in UTC. The fix was to reformat the displayed time so that UTC was readable and understandable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917241[*BZ#1917241*])

Reviewer: ```suggestion
* Previously, *Created date time* was not displayed in a format that a person could read, which made it difficult to understand and use the time shown in UTC. With this release, the displayed time is reformatted so that UTC is readable and understandable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917241[*BZ#1917241*])
```

> * Previously, pod requests and limit calculations in the web console were incorrect. This was a result of not excluding completed pods or init containers. The fix is to exclude pods not needed in the calculation, which improved the accuracy of the results of the web console calculation for pod requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918785[*BZ#1918785*])

Reviewer: ```suggestion
* Previously, pod requests and limit calculations in the web console were incorrect. This was a result of not excluding completed pods or init containers. With this release, pods that are not needed in the calculation are excluded, which improves the accuracy of the results of the web console calculation for pod requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918785[*BZ#1918785*])
```
